recipes/default.rb

```diff
@@ -24,7 +24,14 @@
 
 if EncryptedAttributesCookbook::Helpers.require_build_essential?(gem_version)
   include_recipe 'build-essential'
+
+  # this is necessary because the build-essential recipe isn't loaded until
+  # after the attempt to compile the gem has occurred
+  package 'build-essential' do
+    action :nothing
+  end.run_action(:install)
```

Comment:
hrm.. i am using an older version of the build-essential cookbook due to a conflict with another cookbook my recipe requires. i had tried to set the ['build_essential']['compiletime'] attribute, but looking at the current build-essential cookbook vs the version i'm using, i think that is my problem.

agreed that build-essential should be handling it (or i need to handle it due to the out of date version im using), i'll remove it from this PR.